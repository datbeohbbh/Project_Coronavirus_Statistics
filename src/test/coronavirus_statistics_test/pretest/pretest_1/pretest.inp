10 11
? 5
q 1 5 8
? 4 1
? 4 1
? 4 1
? 6
? 4 1
? 4 1
? 4 1
? 4 1
? 6