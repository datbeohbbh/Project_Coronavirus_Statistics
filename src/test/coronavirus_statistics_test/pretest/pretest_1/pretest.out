Query ? 5 :
The number of safe cities is 10

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

